src/SiteAliasManager.php

```diff
@@ -212,4 +225,9 @@ public function listAllFilePaths($location = '')
     {
         return $this->aliasLoader->listAll($location);
     }
+
+    public function addListener($hook, callable $callback)
+    {
+        $this->dispatcher->addListener($hook, $callback);
```

Comment:
Should `$hook` really be a parameter here? It seems that this should only allow setting our one type of event.

Reply:
For sure, I made it generic with the notion that there might be other event types in the future.
Should it still be called "addListener" then? Would a more specific name be more appropriate then?